app/models/propertyrepresentation/ManageAgentRequest.scala

```diff
@@ -21,7 +21,7 @@ import play.api.data.{Form, Forms}
 import play.api.data.Forms.{longNumber, mapping, optional, text}
 import play.api.libs.json.{Json, OFormat}
 
-case class ManageAgentRequest(agentCode: Long, manageAgentOption: ManageAgentOptions)
+case class ManageAgentRequest(agentName: String, manageAgentOption: ManageAgentOptions)
```

Comment:
I am surprised by this change, are we managing agents by their "name"? but there could be duplicates, no? Couldn't one agent have multiple codes? Then which agent are we managing if we try to identify them by code? Can't they update their Name?

Reply:
we are not managing by name, agentCode is what we pass to the backend. I am passing the name in the url because I need to display it on the page. I didn't want to have to go and fetch it every time (I can do that instead if you guys feel its better to do so). 

I don't think its possible for one agent to have multiple codes.